internal/pkg/utils/testhelper/env.go

```diff
@@ -19,29 +19,39 @@ type EnvProvider interface {
 	MustGet(key string) string
 }
 
-// ReplaceEnvsString replaces ENVs in given string.
-func ReplaceEnvsString(str string, provider EnvProvider) string {
+// ReplaceEnvsStringWithSeparator replaces ENVs in given string with chosen separator.
+func ReplaceEnvsStringWithSeparator(str string, provider EnvProvider, envSeparator string) string {
```

Comment:
A nakonec, jak jsem ti říkal myslím v pátek, že se tam objevil problém v tom že pouštíme funkcionalitu E2E testů v E2E testech a spuštění `cli_test.go` mi chce nahradit placeholdery v `test/cli/template-test/local/in/my-template/v0/tests/` ještě předtím než to má udělat ta operace, tak jsem nakonec změnil formát placeholderu z `%%XXX%%` na `__XXX__` aby se to nebilo a proto tady ta úprava.

Ale jak jsme se bavili, jdu rovnou napsat issue na odstranění závislosti na projektu, čímž by tohle mělo přestat být potřeba. 🙂